<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graph</class>
 <widget class="QDialog" name="Graph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>graph_func</string>
  </property>
  <widget class="QCustomPlot" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>691</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_xy_1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-99</number>
   </property>
   <property name="maximum">
    <number>-1</number>
   </property>
   <property name="value">
    <number>-15</number>
   </property>
   <property name="displayIntegerBase">
    <number>10</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_xy_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>450</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="value">
    <number>15</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_result_1">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>450</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>99</number>
   </property>
   <property name="value">
    <number>15</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spin_result_2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>450</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-99</number>
   </property>
   <property name="maximum">
    <number>-1</number>
   </property>
   <property name="value">
    <number>-15</number>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
